libs/deal-ticket/src/components/deal-ticket/market-selector.tsx

```diff
@@ -196,7 +196,7 @@ export const MarketSelector = ({ market, setMarket, ItemRenderer }: Props) => {
       <div className="relative flex flex-col">
         <div className="relative w-full min-h-[30px]">
           <Input
-            className="h-[30px] w-[calc(100%-20px)] border-none"
+            className="h-[30px] w-[calc(100%-20px)] border-none outline-none"
```

Comment:
question (non-blocking): how anyone will know that this input is in focus?

Reply:
sure, cursor will be visible inside an input

Reply:
moreover we've got there also an arrow button which facilitates a11y:
![Screenshot 2022-09-08 at 15 44 26](https://user-images.githubusercontent.com/338931/189138270-2ebc8209-6421-4324-8c0b-fcff04fc7785.png)